### ISSUE TITLE
What does "modern browsers" mean?

### ISSUE THREAD

**Issue by paulfalgout** [marcuswestin/store.js#219]
I've been trying to find out how this lib defines modern browsers.  
At first glance it looks like modern supports IE8+ and legacy is prior to IE8?

If that's the case, it seems like this library could _very_ safely have the default script in `package.json` be the modern build.

**Comment by mike3run** [comment]
well modern will most likely _at least_ mean: latest version 😄

**Comment by paulfalgout** [comment]
I think in the case of this library it also means really old browsers most people don't support anymore. And legacy means prior to that.

**Comment by Sembiance** [comment]
I'm also wondering specifically which browsers are supported by the 'modern' version.

IE8? IE9?

**Comment by mike3run** [comment]
IE11 + ??

**Comment by mike3run** [comment]
Actually its right on the readme

Tested on IE6+
Tested on iOS 8+
Tested on Android 4+
Tested on Firefox 4+
Tested on Chrome 27+
Tested on Safari 5+
Tested on Opera 11+
Tested on Node (with https://github.com/coolaj86/node-localStorage)

**Comment by Sembiance** [comment]
@mike3run I believe the browsers listed in the readme are for the "legacy" version which has wider support and not the "modern" dist.

**Comment by mike3run** [comment]
True, well reading the sourcecode for like 2 seconds I noted there are 2 files that start with `old`

https://github.com/marcuswestin/store.js/tree/master/storages

Old IE means 6 and 7
Old FF means 6 and 7 as well

So modern browsers would then mean:

Tested on IE8+
Tested on iOS 8+
Tested on Android 4+
Tested on Firefox 8+
Tested on Chrome 27+
Tested on Safari 5+
Tested on Opera 11+
Tested on Node (with https://github.com/coolaj86/node-localStorage)
